medical/gql_mutations.py

```diff
@@ -1,5 +1,5 @@
 import functools
-from gettext import gettext as _
+from django.utils.translation import gettext as _
```

Comment:
Just to be sure - was it working with the old version?

Reply:
i am not sure, i was trying different things to fetch the keys